File: C++/Day8/Rectangle.cpp
#include<iostream>
#include"Shape.h"
#include"Rectangle.h"
using namespace std;

Rectangle::Rectangle(){
	length=0;
	breadth=0;
}
Rectangle::Rectangle(const char* color, double length,double breadth):Shape(color){
	this->length=length;
	this->breadth=breadth;
}

Rectangle::~Rectangle(){};
		
// Getter Methods
double Rectangle::getlength(){
	return length;
}
double Rectangle::getBreadth(){
	return breadth;
}
		
		
// Setter Methods
void Rectangle::setLength(double len){
	this->length=len;
}
void Rectangle::setBreadth(double bt){
	this->breadth=bt;
}
		 
		 
// Calculate Functions
double Rectangle::calcArea(){
	return length*breadth;
}
double Rectangle::calcPerimeter(){
	return 2*(length+breadth);
}
		 
// Display
void Rectangle::display() const{
	cout<<"\n Shape : Rectangle"<<endl;
	Shape::display();
	cout<<"Length : "<<length<<endl;
	cout<<"Breadth : "<<breadth<<endl;
}

File: Java/Day4(Assignment3)/ExercisesDay4/src/Rectangle.java

public class Rectangle {

	double length,breadth;
	public Rectangle()
	{
		
	}
	
	public Rectangle(double length,double breadth)
	{
		this.length=length;
		this.breadth=breadth;
	}

	
	public double getArea()
	{
		return length*breadth;
	}
	public double getLength() {
		return length;
	}

	public void setLength(double length) {
		this.length = length;
	}

	public double getBreadth() {
		return breadth;
	}

	public void setBreadth(double breadth) {
		this.breadth = breadth;
	}
	
	
	
	public static void main(String[] args)
	{
		Rectangle rectangle1=new  Rectangle(5,4);
		Rectangle rectangle2=new  Rectangle(8,5);
		
		System.out.println("Area of rectangle is : "+rectangle1.getArea());
		System.out.println("Area of rectangle is : "+rectangle2.getArea());
				
	}
	
	
}
